### DISCUSSION TITLE
etcdvoter after upgrade from 1.25.14 rke2r1 to 1.26.12

### DISCUSSION THREAD

**Discussion by therus000** [rancher/rke2#5319] [Q&A] [answered]
etcdvoter after upgrade from 1.25.14 rke2r1 to 1.26.12
i tried to update my cluster from 3 nodes (controlplan). 
everything update norm until i updated the firs node
i got status on everynode etcdvoter
the version that i update from 1.25.14 rke2r1
the version that i updated to 1.26.12

**Comment by brandond** [comment, marked as answer]
Yes, we added a status condition for etcd cluster members. It will be true if the node is a voting member of the cluster. If not, the reason and message will describe why.

Is there a problem?

**Comment by therus000** [comment]
is that mean that is this status normal?

![etcdvoter](https://github.com/rancher/rke2/assets/48754322/73856442-e900-487c-8144-e41db282ca5e)

**Comment by brandond** [comment]
Yes.

**Reply by therus000** [reply]
Thank you so much

**Reply by bashar-alasmar** [reply]
Hey mate, why it shows as a warning ?

![image](https://github.com/rancher/rke2/assets/56364147/ccb8d1b7-fb3c-4bd1-a3df-24a8e34a87cb)

**Reply by brandond** [reply]
That's not a warning, its just a status condition. What makes you think it's a warning?

**Reply by bashar-alasmar** [reply]
> That's not a warning, its just a status condition. What makes you think it's a warning?

Thanks for your reply, in Lens which is visual application for Kubernetes(dashboard , metric ..etc)
It shows in the cluster tab in lens as a warning, however its ready and healthy when i checked

**Reply by brandond** [reply]
Then Lens is making incorrect assumptions about the status condition. See the discussion at https://github.com/longhorn/longhorn/issues/7290#issuecomment-1848022567
